### PR TITLE
Connect up existing Project Share Dialog to the Finish Dialog

### PR DESCRIPTION
**Pull request by joshlory**
image

Still need to connect up the right thumbnail and send-to-phone link.

### CODE REVIEW

**Comment by joshlory** [comment]
Not a huge fan of the double-modal, but that's what's spec'd for now 😄.

File: apps/src/templates/FinishDialog.jsx [hunk 1]
```diff
@@ -26,7 +27,7 @@ const styles = {
     left: 0,
     right: 0,
     margin: 'auto',
-    zIndex: 1050,
+    zIndex: 1040,
```
Comment (balderdash) [inline apps/src/templates/FinishDialog.jsx:30]: The backdrop is also at 1040, how about 1045 instead?

Comment (balderdash) [reply apps/src/templates/FinishDialog.jsx:30]: Oh, it's 1040 because it needs to stay under the share dialog backdrop, right?

I guess this works then.

**Comment by joshlory** [comment]
[Finish Dialog: Add “Share” button and associated share UI options](https://trello.com/c/zoE0vG1v/37-finish-dialog-add-share-button-and-associated-share-ui-options)

**Pull request merged**